src/SDKs/StorageSync/Management.StorageSync/Microsoft.Azure.Management.StorageSync.csproj

```diff
@@ -6,14 +6,14 @@
     <PropertyGroup>
         <PackageId>Microsoft.Azure.Management.StorageSync</PackageId>
         <Description>Provides management capabilities for Azure Storage Sync.</Description>
-        <Version>2.8.0-preview</Version>
+        <Version>2.9.0-preview</Version>
```

Comment:
It looks like `2.8.0-preview` was never published. Please do not bump this version